### PR TITLE
Remove one motor from LiftSystem, fix bug in AccelerometerSystem, inv…

### PR DESCRIPTION
**Pull request by jwong101**
…ert lever, change motor controllers, and remove rotation inversion from drive method in WheelSystem

### CODE REVIEW

**Comment by jwong101** [comment]
There was a bug in the AccelerometerSystem where it always showed that the connection was not successful.

File: src/org/impact2585/frc2016/systems/AccelerometerSystem.java [hunk 1]
```diff
@@ -4,6 +4,7 @@
 import org.impact2585.frc2016.Environment;
 import org.impact2585.lib2585.MPU6050;
 
+import edu.wpi.first.wpilibj.I2C;
```
Comment (LoadingPleaseWait) [inline src/org/impact2585/frc2016/systems/AccelerometerSystem.java]: Is this import used?

File: src/org/impact2585/frc2016/systems/ArmSystem.java [hunk 2]
```diff
@@ -138,6 +138,7 @@ public void accessSmartDasboard() {
 	@Override
 	public void run() {
 		double toparmspeed = input.moveTopArm();
+
```
Comment (LoadingPleaseWait) [inline src/org/impact2585/frc2016/systems/ArmSystem.java]: This file shouldn't be edited in this commit.

**Comment by LoadingPleaseWait** [comment]
Can you remove the constant for the lift motor's port number?

**Pull request merged**